pages/manual/contributing.md

```diff
@@ -32,3 +32,13 @@ It is recommended to use [ClangFormat](http://clang.llvm.org/docs/ClangFormat.ht
 
 ### Contributing whole modules
 If you are working on a large Inviwo Module and possibly have dependencies on big external libraries, we have an extra [modules repository](https://github.com/inviwo/modules).
+
+#### Inviwo Marketplace
+Additionally, there is the inviwo marketplace (see Screenshot) that allows developers to share their processors or shaders.
+So after you have generated the code template for a processor by following the [guide for generating processors](https://inviwo.org/manual-devguide-meta.html), you should create a repository on [GitHub](https://github.com/) based on our [template](link-to-template).
```

Comment:
Title:
#### How to add your module to the marketplace
Create a repository ....